File: golf.js
h=parseInt,a=readline().split(' '),b=h(a[0]),c=h(a[1]),d=h(a[2]),e=h(a[3]);for(;;){f='',g='';if(d>b){f='W';d--}if(d<b){f ='E';d++}if(e>c){g='N';e--}if(e<c){g='S';e++}print(g+f)}

File: golf.py
import sys,math;a,b,c,e=[int(i)for i in raw_input().split()]
while 1:
    h='';i=''
    if(c>a):h='W';c+=1
    elif(c<a):h='E';c-=1
    if(e>b):i='N';e-=1
    elif(e<b):i='S';e+=1
    print i+h